recipes/libyang/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout, CMakeDeps
+from conan.tools.files import get, rmdir, copy
+
+required_conan_version = ">=1.53.0"
+
+
+class LibYangConan(ConanFile):
+    name = "libyang"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "YANG data modeling language library"
+    homepage = "https://github.com/CESNET/libyang"
+    topics = ("yang", "bsd", "netconf", "restconf", "yin")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False],
+               "fPIC": [True, False]}
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+
+    def requirements(self):
+        self.requires("pcre2/10.42", transitive_headers=True)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        self.settings.rm_safe("compiler.cppstd")
+        self.settings.rm_safe("compiler.libcxx")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version],
+            destination=self.source_folder, strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["ENABLE_TESTS"] = False
+        tc.variables["ENABLE_VALGRIND_TESTS"] = False
+        tc.variables["ENABLE_STATIC"] = not self.options.shared
+        tc.generate()
+        tc = CMakeDeps(self)
+        tc.generate()
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, "LICENSE", self.source_folder,
+             os.path.join(self.package_folder, "licenses"))
+        cmake = CMake(self)
+        cmake.install()
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+    def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "libyang")
```

Comment:
```suggestion
        self.cpp_info.set_property("cmake_file_name", "LibYANG")
```

https://github.com/CESNET/libyang/blob/master/FindLibYANG.cmake